etl/src/main/java/org/pih/hivmigration/etl/sql/SqlMigrator.java

```diff
@@ -26,23 +26,32 @@
 
     public static final String MIGRATION_PROPERTIES_FILE = "MIGRATION_PROPERTIES_FILE";
 
+    public Properties properties;
+
+    private int rowLimit = -1;
+
+    public void setRowLimit(int rowLimit) {
+        this.rowLimit = rowLimit;
+    }
+
     abstract void migrate();
     abstract void revert();
 
     Properties getMigrationProperties() {
-        Properties p = new Properties();
```

Comment:
All I did was memoize this function, really just so it doesn't produce that log message on every step.